packages/providers/test/fixtures/receipts.ts

```diff
@@ -1,7 +1,7 @@
 import { GqlReceiptType } from '../../src/__generated__/operations';
-import type { GqlReceiptFragmentFragment } from '../../src/__generated__/operations';
+import type { GqlReceipt } from '../../src/__generated__/operations';
```

Comment:
Why `GqlReceipt` instead of `GqlReceiptDryRunFragmentFragment`, following the previous implementation?

```suggestion
import type { GqlReceiptDryRunFragmentFragment } from '../../src/__generated__/operations';
```